### DISCUSSION TITLE
get trouble with start exporter

### DISCUSSION THREAD

**Discussion by mcl116cn** [ClusterLabs/ha_cluster_exporter#233] [General]
i've built up a 3-node-cluster with coroysnc and pacemaker
but when i startup the ha_cluster_exporter the log shows error:

could not initialize 'drbd' collector : '/sbin/drbdsetup' does not exist 

which means must i have to built a drbd cluster?

**Comment by stefanotorresi** [comment]
The exporter needs to be able to initialize at least one collector. As long as one of the others works, you can safely ignore that message. The process will only fail hard if it can't initialize _any_ collector.
